### ISSUE TITLE
Move parameterisations into folder

### ISSUE THREAD

**Issue by milankl** [SpeedyWeather/SpeedyWeather.jl#183]
Now as there are more and more parameterisations coming in I would suggest to move every parameterization-related to `src/parameterizations` eventually this could even become it's own module, but I don't think this is needed now.

At the moment I think to move in there
- column_variables.jl
- convection.jl
- large_scale_condensation.jl
- thermodynamics.jl
- shortwave_radiation.jl
- longwave_radiation.jl

so basically everything that is called within `parameterization_tendencies!` (but I would leave that one as it defines the interface on the dynamics side).

@white-alistair @dmey
